cli/__test__/index.gr

```diff
@@ -1,4 +1,4 @@
-/* grainc-flags --no-gc --no-bulk-memory --no-wasm-tail-call */
+@compileFlags("--no-gc", "--no-bulk-memory", "--no-wasm-tail-call")
```

Comment:
I was considering having the `-`/`--` be included automatically but one problem I see with that is that if someone wanted to do something like `--memory-base 0x400` they may accidentally do `@compile("memory-base", "0x400")` which wouldn't work. Of course in that situation you _could_ allow `@compile("memory-base 0x400")` and then split all of the arguments by spaces to get `--memory-base` and `0x400` be separate arguments rather than one `'--memory-base 0x400'` argument, but that itself creates additional issues e.g. using directories with spaces in their names. What do you think?

Reply:
As I mentioned in a different comment, I think these should require `=` for compile options that need a value

Reply:
Ok so should a `-` be injected if whatever comes before the `=` character is only one character and `--` otherwise?

Reply:
Why are you "injecting"? Maybe you should consider trimming the flags coming from the CLI 🤔

Reply:
True, just that prepending it into the string seems much easier to implement within the existing codebase